### PR TITLE
Cleanup colors

### PR DESCRIPTION
**Pull request by franzheidl**
* Rename colors according to the names used in our Figma files
* mapping to SAP web brand color names are in comments
* update / change all occurences / references
* bump to 1.4.2

### CODE REVIEW

File: libs/juno-ui-components/tailwind.config.js [hunk 1]
```diff
@@ -104,53 +104,54 @@ module.exports = {
...
-        6: withOpacity("--color-sap-blue-dark-1-raw"),
-        7: withOpacity("--color-sap-blue-dark-raw"),
+        1: withOpacity("--color-sap-blue-1-raw"),
+        2: withOpacity("--color-sap-blue-2-raw"),
+        3: withOpacity("--color-sap-blue-3-raw"),
+        4: withOpacity("--color-sap-blue-4-raw"),
+        5: withOpacity("--color-sap-blue-5-raw"),
+        6: withOpacity("--color-sap-blue-6-raw"),
+        7: withOpacity("--color-sap-blue-7-raw"),
         DEFAULT: withOpacity("--color-sap-blue-raw"),
```
Comment (edda) [inline libs/juno-ui-components/tailwind.config.js:114]: the default value needs to be adjusted to `--color-sap-blue-5-raw`

File: libs/juno-ui-components/tailwind.config.js [hunk 2]
```diff
@@ -104,53 +104,54 @@ module.exports = {
...
-        6: withOpacity("--color-sap-gold-dark-raw"),
-        7: withOpacity("--color-sap-gold-dark-1-raw"),
+        1: withOpacity("--color-sap-gold-1-raw"),
+        2: withOpacity("--color-sap-gold-2-raw"),
+        3: withOpacity("--color-sap-gold-3-raw"),
+        4: withOpacity("--color-sap-gold-4-raw"),
+        5: withOpacity("--color-sap-gold-5-raw"),
+        6: withOpacity("--color-sap-gold-6-raw"),
+        7: withOpacity("--color-sap-gold-7-raw"),
         DEFAULT: withOpacity("--color-sap-gold-raw"),
```
Comment (edda) [inline libs/juno-ui-components/tailwind.config.js:124]: default value needs adjustment (see above)

File: libs/juno-ui-components/tailwind.config.js [hunk 3]
```diff
@@ -104,53 +104,54 @@ module.exports = {
...
-        6: withOpacity("--color-sap-purple-dark-raw"),
-        7: withOpacity("--color-sap-purple-dark-1-raw"),
+        1: withOpacity("--color-sap-purple-1-raw"),
+        2: withOpacity("--color-sap-purple-2-raw"),
+        3: withOpacity("--color-sap-purple-3-raw"),
+        4: withOpacity("--color-sap-purple-4-raw"),
+        5: withOpacity("--color-sap-purple-5-raw"),
+        6: withOpacity("--color-sap-purple-6-raw"),
+        7: withOpacity("--color-sap-purple-7-raw"),
         DEFAULT: withOpacity("--color-sap-purple-raw"),
```
Comment (edda) [inline libs/juno-ui-components/tailwind.config.js:134]: default value needs adjustment (see above)

File: libs/juno-ui-components/tailwind.config.js [hunk 4]
```diff
@@ -104,53 +104,54 @@ module.exports = {
...
-        6: withOpacity("--color-sap-green-dark-raw"),
-        7: withOpacity("--color-sap-green-dark-1-raw"),
+        1: withOpacity("--color-sap-green-1-raw"),
+        2: withOpacity("--color-sap-green-2-raw"),
+        3: withOpacity("--color-sap-green-3-raw"),
+        4: withOpacity("--color-sap-green-4-raw"),
+        5: withOpacity("--color-sap-green-5-raw"),
+        6: withOpacity("--color-sap-green-6-raw"),
+        7: withOpacity("--color-sap-green-7-raw"),
         DEFAULT: withOpacity("--color-sap-green-raw"),
```
Comment (edda) [inline libs/juno-ui-components/tailwind.config.js:144]: default value needs adjustment (see above)

File: libs/juno-ui-components/tailwind.config.js [hunk 5]
```diff
@@ -104,53 +104,54 @@ module.exports = {
...
-        7: withOpacity("--color-sap-red-dark-1-raw"),
+        1: withOpacity("--color-sap-red-1-raw"),
+        2: withOpacity("--color-sap-red-2-raw"),
+        3: withOpacity("--color-sap-red-3-raw"),
+        4: withOpacity("--color-sap-red-4-raw"),
+        5: withOpacity("--color-sap-red-5-raw"),
+        6: withOpacity("--color-sap-red-6-raw"),
+        7: withOpacity("--color-sap-red-7-raw"),
+        8: withOpacity("--color-sap-red-8-raw"),
         DEFAULT: withOpacity("--color-sap-red-raw"),
```
Comment (edda) [inline libs/juno-ui-components/tailwind.config.js:155]: default value needs adjustment (see above)

**Review by edda — CHANGES_REQUESTED** [review]

**Comment by franzheidl** [comment]
Default colors fixed.

**Review by edda — APPROVED** [review]

**Pull request merged**